zrml/asset-router/src/tests/inspect.rs

```diff
@@ -21,12 +21,12 @@ use super::*;
 use frame_support::traits::tokens::fungibles::Inspect;
 
 fn test_helper(asset: Assets, initial_amount: <Runtime as crate::Config>::Balance) {
-    assert!(AssetRouter::asset_exists(asset));
     assert_ok!(<AssetRouter as orml_traits::MultiCurrency<AccountId>>::deposit(
         asset,
         &ALICE,
         initial_amount
     ));
+    assert!(AssetRouter::asset_exists(asset));
```

Comment:
What's the purpose of this change?

Reply:
If that assertion is invoked before the deposit, the `Currencies` case will error, because in the `Currencies` case the asset is considered as existing when `total_issuance > 0`.